web/html/src/manager/content-management/shared/api/use-lifecycle-actions-api.ts

```diff
@@ -49,8 +49,13 @@ const useLifecycleActionsApi = (props: Props): returnUseProjectActionsApi => {
       setIsLoading(true);
 
       const apiUrl = getApiUrl(props.resource, props.nestedResource, id);
-      const networkMethod: NetworkMethod = networkAction[action] || networkAction["get"];
-      const networkRequest = networkMethod(apiUrl, JSON.stringify(actionBodyRequest), "application/json");
```

Comment:
There was a bug here, where GET requests would also be called with the following signature:  

```ts
get(url, payload, mimeType)
```

However, their actual signature is:

```ts
get(url, mimeType)
```

This only accidentally worked because GET requests have a default value for `contentType: string = "application/json"` (other requests don't have this default value).  
The below refacto aims to fix this problem.